USFMConverter/Core/CoreConverter.cs

```diff
@@ -2,29 +2,36 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
+using System.Net;
+using System.Runtime.InteropServices.ComTypes;
 using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
 using USFMConverter.Core.ConstantValue;
 using USFMConverter.Core.Data;
 using USFMConverter.Core.Render;
 using USFMConverter.Core.Util;
 using USFMConverter.UI;
+using USFMToolsSharp.Models.Markers;
 
 namespace USFMConverter.Core
 {
     public class CoreConverter
     {
         public static ICollection<string> supportedExtensions = new List<string> {
-            ".usfm", ".txt", ".sfm"
```

Comment:
Good questions, it never actually did anything since it couldn't understand the JSON wrapped USFM that writer uses it wasn't enough just to add .txt as an acceptable file format. I removed this here because BTTWriterLib will handle this directly. So instead of looking for .txt files we now find the manifest.json and use the loader to find all the other files to load.